cpp/ql/src/Critical/MissingCheckScanf.qhelp

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+
+<overview>
+<p>
+This query finds calls of <tt>scanf</tt>-like functions with missing or
+improper return-value checking.
+</p>
+<p>
+Specifically, the query flags uses of variables that may have been modified by
+<tt>scanf</tt> and subsequently are used without being guarded by a correct
+return-value check. A proper check is one that asserts the corresponding
```

Comment:
Consider replacing "asserts" with another word ("ensures"?), as some developers may read "asserts" as a reference to the `assert` macro and miss the point of this sentence.